src/gui/newwizard/pages/AbstractSetupWizardPage.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <QWidget>
+
+namespace OCC::Wizard {
+
+class AbstractSetupWizardPage : public QWidget
+{
+    Q_OBJECT
+
+public:
+    ~AbstractSetupWizardPage() override;
+
+public Q_SLOTS:
```

Comment:
Why a slot when it is directly called?
Maybe rather define a signal that can be subscribed to instead of requiring to define an empty virtual function?

Reply:
I first had a default implementation in place, but that was only useful for a single class. All other classes need to perform custom setup code.

I think this solution is more reliable, because the child class does not need to know about the parent class.

Reply:
Then unslotify it, its not used as a slot.
A signal would still be more _Qt_